File: Code 3 Callouts/Code 3 Callouts/Models/IPoliceIncident.cs
using Rage;
using Stealth.Plugins.Code3Callouts.Models.Peds;
using Stealth.Plugins.Code3Callouts.Util.Audio;
using System.Collections.Generic;
using static Stealth.Plugins.Code3Callouts.Util.Audio.AudioDatabase;

namespace Stealth.Plugins.Code3Callouts.Models
{

    internal interface IPoliceIncident
	{

		Vector3 GetRandomSpawnPoint(float pMin, float pMax);
		PedBase GetPed(string pName);
		Vehicles.Vehicle GetVehicle(string pName);

		int RadioCode { get; set; }
		List<DISPATCH.CRIMES> CrimeEnums { get; set; }

		Vector3 SpawnPoint { get; set; }
		List<PedBase> Peds { get; set; }

		List<Vehicles.Vehicle> Vehicles { get; set; }
	}

}

File: Code 3 Callouts/Code 3 Callouts/Models/IPoliceIncident.vb
Imports Rage
Imports Stealth.Plugins.Code3Callouts.Models.Peds
Imports Stealth.Plugins.Code3Callouts.Util.Audio

Namespace Models

    Public Interface IPoliceIncident

        Function GetRandomSpawnPoint(ByVal pMin As Single, ByVal pMax As Single) As Vector3
        Function GetPed(ByVal pName As String) As PedBase
        Function GetVehicle(ByVal pName As String) As Vehicles.Vehicle

        Property RadioCode As Integer
        Property CrimeEnums As List(Of DISPATCH.CRIMES)
        Property SpawnPoint As Vector3

        Property Peds As List(Of PedBase)
        Property Vehicles As List(Of Vehicles.Vehicle)

    End Interface

End Namespace